### PR TITLE
Beomki Yeo, new team member

### PR DESCRIPTION
**Pull request by beomki-yeo**
I am a new postdoc research of UC berkeley (PI: heather gray)

### CODE REVIEW

File: _data/people/beomki-yeo.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: Beomki Yeo
+shortname: beomki-yeo
+title: Post-doctoral researcher
+active: true
+institution: University of California, Berkeley
+website:
+photo: /assets/images/team/beomki-yeo.png
+e-mail: beomki.yeo@berkeley.edu
+presentations:
```
Comment (matthewfeickert) [inline _data/people/beomki-yeo.yml]:
```suggestion
```

@beomki-yeo :wave: Welcome to the team! Can you please rebase this PR off of `master` to pick up Carlo's PR and avoid the merge conflict and then also remove this presentations field (I think that it being empty is causing the CI to fail).

**Review by matthewfeickert — CHANGES_REQUESTED** [review]

File: _data/people/beomki-yeo.yml (hunk 1 above)
Comment (beomki-yeo) [reply _data/people/beomki-yeo.yml]: Done! but CI is still failing :(((

**Comment by matthewfeickert** [comment]
Hm, CI is failing with 

```console
/home/runner/work/iris-hep.github.io-source/iris-hep.github.io-source/_plugins/checkusers.rb:35:in `block in generate': beomki-yeo is not listed in a university and hidden is not True (StandardError)
```

I'm a bit confused by this given that the YAML looks fine, so I'll have to look at it again tomorrow (unless @henryiii can see an obvious error that I'm missing).

**Comment by matthewfeickert** [comment]
Oh the CI is failing as you didn't add yourself to https://github.com/iris-hep/iris-hep.github.io-source/blob/1bdf20a661ae8c74cbdcfa260d343c499d0e2224/_data/universities/berkeley.yml

**Review by matthewfeickert — APPROVED** [review]

**Pull request merged**